app-editors/bvi/bvi-1.4.0.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Display-oriented editor for binary files, based on the vi texteditor"
+HOMEPAGE="http://bvi.sourceforge.net/"
+SRC_URI="mirror://sourceforge/bvi/${P}.src.tar.gz"
+
+LICENSE="GPL-2"
```

Comment:
Could you verify the license, please? In particular, look at copyright notices on top of source files. If they state 'version 2 or (at your blah blah) newer version', then it should be `GPL-2+`.

Reply:
Interesting. The website doesn’t say “or later”, but the source files do. So I guess it’s 2+. 1407b64e9d00